src/main/scala/com/actian/spark_vector/vector/VectorOps.scala

```diff
@@ -80,9 +93,21 @@ trait VectorOps {
       selectColumns: String = "*",
       whereClause: String = "",
       whereParams: Seq[Any] = Nil): RDD[Row] = {
-      Vector.unloadVector(sc, vectorProps, targetTable, tableMetadataSchema,
+      Vector.unloadVector(sc, targetTable, vectorProps, tableMetadataSchema,
```

Comment:
since we're changing param names:
targetTable <- table
tableMetadataSchema <- tableCoumnMetadata

(or the other way around, but stick to one of them :+1: )